stdlib/public/SDK/CoreGraphics/CGFloat.swift.gyb

```diff
@@ -136,9 +136,9 @@ public struct CGFloat {
     return CGFloat(NativeType.signalingNaN)
   }
   
-  @available(*, deprecated, renamed: "nan")
+  @available(*, unavailable, renamed: "nan")
   public static var quietNaN: CGFloat {
-    return CGFloat(NativeType.nan)
+    fatalError("unavailable")
```

Comment:
Unfortunately we can't use `Builtin` outside of the core library without changing the build system.